.github/renovate.json

```
@@ -0,0 +1,17 @@
+{
+	"$schema": "https://docs.renovatebot.com/renovate-schema.json",
+	"extends": ["github>sapphiredev/readme:sapphire-renovate"],
+	"labels": ["Meta: Dependencies"],
+	"npm": {
+		"packageRules": [
+			{
+				"matchPackagePatterns": ["tslib", "colorette", "i18next", "discord-api-types"],
```

```suggestion
				"matchPackagePatterns": ["discord-api-types"],
```

tslib was there because of typeorm
colorette and i18next I don't remember